File: Service/Models/TournamentModel.cs
namespace TecmoTourney.Models
{
    public class TournamentModel
    {
        public int TournamentId { get; set; }
        public string Name { get; set; } = string.Empty;
        public string BracketData { get; set; } = string.Empty;
        public string BracketImage { get; set; } = string.Empty;
        public TournamentStatus Status { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
    }
}

File: Client/tecmo-tourney/src/app/core/models/tournament.model.ts
import { TournamentStatus } from 'src/app/enums';

export interface ITournament {
    tournamentId: number;
    name: string;
    location: string;
    status: TournamentStatus;
    startDate: string;
    endDate: string;
    tournamentBracket: any;
  }